packages/page-broker/src/Overview/Filters.tsx

```diff
@@ -16,31 +17,31 @@ const StyledDiv = styled.div`
 `;
 
 interface Props {
-  workLoad?: CoreWorkloadType[];
-  onFilter: (data: CoreWorkloadType[]) => void
+  data: CoreInfo[];
+  onFilter: (data: CoreInfo[]) => void
 }
 
-const filterLoad = (parachainId: string, load: CoreWorkloadType[] | CoreWorkplanType[], workloadCoreSelected: number): CoreWorkloadType[] | CoreWorkplanType[] => {
+const filterLoad = (parachainId: string, data: CoreInfo[], workloadCoreSelected: number): CoreInfo[] => {
   if (parachainId) {
-    return load?.filter(({ info }) => info.task === parachainId);
+    return data.filter(({ workload }) => !!workload?.filter(({ info }) => info.task === parachainId).length);
```

Comment:
Wouldn't `!!workload` here just return a boolean?

Reply:
Yes, that is exactly what I am going for :) Coz I need to see if the is  a workload with parachainId that I am looking for :) Then I can filter out objects of CoreInfo 

```
export interface CoreInfo {
  core: number,
  workload: CoreWorkloadType[] | undefined,
  workplan: CoreWorkplanType[] | undefined
}
```